catalog/spatial/wfs/2.0.0/spatial-wfs-v2_0_0-source/src/main/java/org/codice/ddf/spatial/ogc/wfs/v2_0_0/catalog/source/WfsSource.java

```diff
@@ -341,47 +342,53 @@ public void refresh(Map<String, Object> configuration) {
 
   /* This method should only be called after all properties have been set. */
   private void createClientFactory() {
+    ClientFactoryBuilder<Wfs> clientFactoryBuilder = clientFactory.getClientFactoryBuilder();
```

Comment:
❓ Can we make this a completely fluent api? It would be nice if we didn't have to see the builder at all. I'm imagining something like:
```
ClientFactory clientFactory = clientFactory.builder()
    .prop1()
    .prop2()
    .build()
```

Reply:
I mean, you don't technically have to see it right now, I just chose to put it on two lines.

Reply:
Oh lol. Well I guess then just take this as a statement of my preference of using the API fluently then 😄